src/ProductConstructionService/ProductConstructionService.Api/Metrics/TelemetryRecorder.cs

```diff
@@ -2,40 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics;
-using System.Diagnostics.Metrics;
 using Microsoft.ApplicationInsights;
 using ProductConstructionService.Api.Queue.Jobs;
-using ProductConstructionService.ServiceDefaults;
 
 namespace ProductConstructionService.Api.Metrics;
 
-public class TelemetryRecorder(IMeterFactory meterFactory, ILogger<TelemetryRecorder> logger, TelemetryClient telemetryClient) : ITelemetryRecorder
+public class TelemetryRecorder(ILogger<TelemetryRecorder> logger, TelemetryClient telemetryClient) : ITelemetryRecorder
 {
     private readonly ILogger<TelemetryRecorder> _logger = logger;
-    private readonly Meter _meter = meterFactory.Create(MetricConsts.JobMeterName);
-    private readonly Dictionary<string, Histogram<long>> _histograms = [];
     private readonly TelemetryClient _telemetryClient = telemetryClient;
 
     public ITelemetryScope RecordJob(Job job)
     {
-        var instrumentName = $"job.{job.Type}";
-
-        if (!_histograms.TryGetValue(instrumentName, out var histogram))
-        {
-            var histogramName = $"job.{job.Type}.duration";
-
-            histogram = _meter.CreateHistogram<long>(histogramName);
-            _histograms.Add(instrumentName, histogram);
-        }
-        return new TelemetryScope(job.Type, histogram, _logger, _telemetryClient);
+        return new TelemetryScope($"{job.Type} completed", _logger, _telemetryClient, []);
```

Comment:
This should be again something like before:
`job.jobType`

So that we can filter those on a common prefix.

`completed` is extra, we know we only send the telemetry at the end.

Reply:
done